Gemfile

```diff
@@ -2,4 +2,4 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in bitex_bot.gemspec
 gemspec
-gem 'bitstamp', github: 'bitex-la/bitstamp'
+gem 'bitstamp', github: 'bitex-la/bitstamp', branch: 'currency_pair_orders'
```

Comment:
`currency_pair_orders` is merged, let's revert this bit.

Reply:
👍